IOTConnectSDK/iotconnect-sdk_2_1/build.gradle

```diff
@@ -71,4 +95,33 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
-}
\ No newline at end of file
+}
+
+
+afterEvaluate {
+    publishing {
+        publications {
+
+            brokerazRelease(MavenPublication) {
+                from components.brokerazRelease
+
+                groupId = 'com.github.sw-ms-imranbhadelia'
+                artifactId = 'iotconnect-sdk'
+                version = '12.2.1-azTest'
+
+            }
+
+            /*brokerawsRelease(MavenPublication) {
```

Comment:
remove unwanted code

Reply:
@sw-ot-deveshmevada 
These code is used to generate library for different product flavors (AZ/AWS). At a time one is commented and other is used for creating different library according to product flavors (AZ/AWS)

Reply:
Can we do like this?
`android {

    flavorDimensions "version"

    productFlavors {
        free {
            dimension "version"
            // Customize properties for the 'free' flavor
            applicationId "com.example.free"
            versionCode 1
            versionName "1.0-free"
        }

        pro {
            dimension "version"
            // Customize properties for the 'pro' flavor
            applicationId "com.example.pro"
            versionCode 1
            versionName "1.0-pro"
        }

        demo {
            dimension "version"
            // Customize properties for the 'demo' flavor
            applicationId "com.example.demo"
            versionCode 1
            versionName "1.0-demo"
        }
    }

}
`